apps/daffio/src/app/core/sidebar/containers/sidebar-viewport/sidebar-viewport.component.html

```diff
@@ -1,6 +1,9 @@
+<div></div>
```

Comment:
Do we need this div?

Reply:
Addressed in 5de79c050489de6e9ea9890dcbd9152c51a22c04